interop/test_utils.go

```diff
@@ -454,6 +454,41 @@ func DoCancelAfterFirstResponse(tc testpb.TestServiceClient) {
 	}
 }
 
+// DoStatusCodeAndMessage checks that the status code is propagated back to the client.
+func DoStatusCodeAndMessage(tc testpb.TestServiceClient) {
+	var code int32 = 2
+	msg := "test status message"
+        expectedErr := grpc.Errorf(codes.Code(code), msg)
+	respStatus := &testpb.EchoStatus{
+		Code:    proto.Int32(code),
+		Message: proto.String(msg),
+	}
+	// Test UnaryCall.
+	req := &testpb.SimpleRequest{
+		ResponseStatus: respStatus,
+	}
+	if _, err := tc.UnaryCall(context.Background(), req); err.Error() != expectedErr.Error() {
+		grpclog.Fatalf("UnaryCall RPC returned error %v, want %v", err, expectedErr)
+	}
+	// Test FullDuplexCall.
+	stream, err := tc.FullDuplexCall(context.Background())
+	if err != nil {
+		grpclog.Fatalf("%v.FullDuplexCall(_) = _, %v, want <nil>", tc, err)
+	}
+	stream_req := &testpb.StreamingOutputCallRequest{
+		ResponseStatus: respStatus,
+	}
+	if err := stream.Send(stream_req); err != nil {
+		grpclog.Fatalf("stream %v.Send(%v) = %v, want <nil>", stream, stream_req, err)
+	}
+	if err := stream.CloseSend(); err != nil {
+		grpclog.Fatalf("stream %v.CloseSend() = %v, want <nil>", stream, err)
+	}
+	if _, err = stream.Recv(); err.Error() != expectedErr.Error() {
+		grpclog.Fatalf("stream %v.Recv() returned error %v, want %v", err, expectedErr)
```

Comment:
Remove "stream" from fatalf. And you missed the variable for the first `%v`
`grpclog.Fatalf("%v.Recv() returned error %v, want %v", stream, err, expectedErr)`